samples/PSFScriptingSample/readme.md

```diff
@@ -0,0 +1,13 @@
+# Scripting sample
+This solution contains example scripts and configurations to make an application to see how scripting works in the Package Support Framework.
+
+## How to make the package?
+To make and install the sample application all you need to do is rebuild the solution from inside Visual Studio.  The build system will package the app and install it on your computer.
+
+The applications that are installed are
+
+  * PSFScriptingSampleOne
+  * PSFScriptingSampleTwo
+  
+## Modifying the sample.
+Feel free to modify the config.json file to see how all the script objects work and how they affect the script execution.
```

Comment:
>Feel [](start = 0, length = 4)

Maybe not for this PR, but this could be a great place to list out an exercise guiding a user on specific changes to make.